presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -951,6 +955,16 @@ public void testParseDuration()
         assertInvalidFunction("parse_duration('abc')", "duration is not a valid data duration string: abc");
     }
 
+    @Test
+    public void testToMilliseconds()
```

Comment:
you can be more specific with the method name to include that you are converting `INTERVAL_DAY_TO_SECOND`